src/index.html

```diff
@@ -6,8 +6,33 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <title>Product cards</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Product cards</h1>
+    <div class="card" data-qa="card" >
+      <img src="images/imac.jpeg" alt="mac" class="card__image">
+      <h3 class="card__title">Моноблок <span class="text-up">APPLE A</span>1419 iMac 27&Prime; Retina
+        5<span class="text-up">K</span> (MNED2UA/A)</h3>
+      <p class="card__code">
+        <span>Код товара: 195434</span>
+      </p>
+      <div class="star__review">
+        <div class="card__stars">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+        <p class="card__review">Отзывов: 5</p>
+      </div>
+      <p class="card__price">Цена:
```

Comment:
```suggestion
      <p class="card__price">
        <span>Цена:</span>
```
если в блоке более одного элемента, лучше оборачивать каждый в тег